### ISSUE TITLE
Add a "preview on hover" feature like Wikipedia Quick Hints

### ISSUE THREAD

**Issue by vickychijwani** [vickychijwani/quickwiki#10]
There's another Chrome extension, [Wikipedia Quick Hints][1], that lets users preview the opening paragraph of an article simply by *hovering* over the link. We should consider adding a feature like that. Seems quite handy to me.

[1]: https://chrome.google.com/webstore/detail/wikipedia-quick-hints/ldnhgfghebflgcndlbppfanbchpgmkna

**Comment by koustuvsinha** [comment]
Yes we can add a feature like that. Key point is to display the information in a visually pleasing style.
The extension adds a small question mark at every link, which gets too cluttered in hover modals. We can try to make it more simpler and effective!

**Comment by koustuvsinha** [comment]
Logic to implement this feature : We have to extract the first paragraph of a wiki page as done by the above extension. But this will not work for Wikia.com pages. Suggest some alternative

**Comment by koustuvsinha** [comment]
Possible workaround for Wikia.com : upon further investigation of their source, it seems that they have `meta property="og:description"` tag, which has a short description on the page. We can display that

**Comment by koustuvsinha** [comment]
Currently implementing "Popup on hover" only for wikipedia because of the above issue. The `meta property="og:description"` tag contains too little information to show, so not going to explore that
